migrations/20170301153034_AddAvgLastPrice.up.sql

```diff
@@ -1 +1 @@
-ALTER TABLE Stocks ADD COLUMN avgLastPrice int(11);
+ALTER TABLE Stocks ADD COLUMN avgLastPrice bigint(11);
```

Comment:
I feel it's better to create a new migration instead of modifying previous ones. The advantage is you don't have to be bothered about _how_ you got to the current schema. Might seem messy but makes sure that you don't have to worry about what happened in a previous migration.

Reply:
I added this here because it's something that you left out while making this change : #244